Tools/CISupport/ews-build/master.cfg

```diff
@@ -60,16 +60,21 @@ if not is_test_mode_enabled:
         },
     )
 
-    credentials = load_password('EWS_CREDENTIALS')
-    if not credentials:
-        print('EWS credentials not found. Please ensure EWS_CREDENTIALS is configured either in env variables or in passwords.json')
+    GITHUB_CLIENT_ID = load_password('GITHUB_CLIENT_ID')
+    GITHUB_CLIENT_SECRET = load_password('GITHUB_CLIENT_SECRET')
+    if (not GITHUB_CLIENT_ID) or (not GITHUB_CLIENT_SECRET):
+        print('EWS credentials not found. Please ensure GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET are configured either in env variables or in passwords.json')
         sys.exit(1)
+
     # See https://docs.buildbot.net/current/manual/configuration/www.html#example-configs
     authz = util.Authz(
-        allowRules=[util.AnyControlEndpointMatcher(role="admin")],
-        roleMatchers=[util.RolesFromEmails(admin=list(credentials.keys()))]
+        allowRules=[util.AnyEndpointMatcher(role='Administrators', defaultDeny=False),
+                    util.AnyEndpointMatcher(role='Bot Watchers', defaultDeny=False),
+                    util.RebuildBuildEndpointMatcher(role='Contributors'),
```

Comment:
`Administrators` and `Bot Watchers` are currently manually configured (which I think is appropriate). `Contributors` is derived from `contributors.json`. At the moment, this is a somewhat manual process (there is a script I occasionally run). We should have `commits.webkit.org` own this, but we would need a bot account that is an owner of the WebKit organization, which has been my hesitation. I think I have an idea of how to making this palatable (given th super-powers an organization owner ends up having), but @aj062 need to take that discussion offline.